app/assets/stylesheets/hubstats/bootstrap.css

```diff
@@ -654,7 +654,8 @@ blockquote .small:before {
   content: '\2014 \00A0';
 }
 .blockquote-reverse,
-blockquote.pull-right {
+blockquote.pull-right,
+blockquote.deploy-right {
```

Comment:
Are these actually needed since we determined here that `pull` is a bootstrap class and not in reference to a "pull request"?